proc_CPMG.py

```diff
@@ -62,10 +62,10 @@ def print_fun(x, f, accepted):
     fl.image(s.imag)
     #}}}
     #{{{select echo decay fit function
-    s.ift('t2')
+    #s.ift('t2')
     s = s['t2':(0,None)]
     s['t2',0] *= 0.5
-    s.ft('t2')
+    #s.ft('t2')
```

Comment:
why are these commented out -- it looks like you're slicing out the FID -- why would you not do that in the time domain?

Reply:
this is outdated

Reply:
in commit bbad, they were commented out again

Reply:
so I'm saying that I think this needs to be fixed? that they need to be uncommented for reasons noted above.

Reply:
Okay I commented these back in as you suggested and it still runs nicely